### PR TITLE
[DNM] jewel: rgw: radosgw-admin zonegroup get and zone get return defaults when there i…

### PR DESCRIPTION
**Pull request by pdvian**
http://tracker.ceph.com/issues/22211

### CODE REVIEW

**Comment by pdvian** [comment]
@cbodley @lvshanchun @smithfarm

File: qa/tasks/radosgw_admin.py [hunk 1]
```diff
@@ -1026,9 +1026,51 @@ def task(ctx, config):
...
     assert len(out['placement_pools']) == orig_placement_pools + 1
+
+    zonecmd = ['zone', 'placement', 'rm',
+	'--rgw-zone', 'default',
+	'--placement-id', 'new-placement']
+
+    (err, out) = rgwadmin(ctx, client, zonecmd, check_status=True)
+
+    # TESTCASE 'zonegroup-info', 'zonegroup', 'get', 'get zonegroup info', 'succeeds'
+    (err, out) = rgwadmin(ctx, client, ['zonegroup', 'get'], check_status=True)
```
Comment (cbodley) [inline qa/tasks/radosgw_admin.py]: this was the last line in the original diff - i'm not sure we want to include the remainder

Comment (pdvian) [reply qa/tasks/radosgw_admin.py]: @cbodley made the changes. Kindly review it.

**Comment by pdvian** [comment]
retest this please

**Comment by smithfarm** [comment]
test this please

**Comment by smithfarm** [comment]
test this please

**Comment by smithfarm** [comment]
jenkins test this

**Comment by smithfarm** [comment]
retest this please

**Comment by smithfarm** [comment]
test this please

**Comment by smithfarm** [comment]
retest this please

**Comment by smithfarm** [comment]
jenkins re-test this please

**Comment by smithfarm** [comment]
Hit transient make check failure in test/osd/osd-recovery-scrub.sh - tracking at http://tracker.ceph.com/issues/23105

Jenkins, re-test this please

**Comment by smithfarm** [comment]
removed "jewel-batch-1" because this PR looks like it might have caused the failures in http://pulpito.ceph.com:80/smithfarm-2018-02-23_19:41:10-rgw-wip-jewel-backports-distro-basic-smithi/

**Comment by smithfarm** [comment]
Upon taking a closer look, I'm pretty sure the test failures are caused by this PR. All the failures look like this: "Command failed on smithi204 with status 2: 'adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage radosgw-admin --log-to-stderr --format json -n client.0 zone placement rm --rgw-zone default --placement-id new-placement'"

And it seems to me this command was added to the test by this PR.

**Comment by smithfarm** [comment]
Jewel is EOL

**Pull request closed without merging**